### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project a hambuger button is built so that that when the user clicks on the menu bar or hamburger button displays the different menu options that is portfolio , about and contact and when the user clicks on each of the nav links it takes them to the page in the website
## Built With

- HTML
- CSS
-javascript
- Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,15 +10,31 @@
...
+        <img src="Resources/Images/markx.png" alt="x icon">
+    </div>
+
+      <ul class="popupbars">
+        <li><a href="#">Portoflio</a></li>
+        <li ><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav> -->
+
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

File: index.html [hunk 2]
```diff
@@ -51,6 +67,33 @@ <h2 class="subhead">I'am a Software Developer</h2>
...
+                <button>See Live 
+                    <img src="Resources/Images/searchbutt.png" alt="search icon"></button>
+            </div>
+            <div class="multi-butone">
+                <button>See Source 
+                    <img src="Resources/Images/whiteangelist.png" alt="github icon"></button>
+            </div>
+        </div>
+    </section> -->
+
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Ngufor-emmanuella) [reply index.html]: its done thank you

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality and the design✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank you @Reem-lab

**Pull request merged**